src/pages/library/Datasets.js

```diff
@@ -456,7 +455,7 @@ const Datasets = () => {
     }),
     catalogShowing ? h(Browser) :
       div({ role: 'main', style: styles.content }, [
-      // Put datasets in alphabetical order
+        // Put datasets in alphabetical order
```

Comment:
Is this accurate? Should rareX be listed before target?

Reply:
Thank you for that catch!
Changed in [67b2476](https://github.com/DataBiosphere/terra-ui/pull/3374/commits/67b2476c0bcd91695324baae23a9ac29bcfd06f5).